pyproject.toml

```diff
@@ -1,15 +1,16 @@
 [tool.poetry]
-name = "openrelik-worker-TEMPLATEWORKERNAME"
+name = "openrelik-worker-cloud-logs"
 version = "0.1.0"
-description = "Description of the worker"
-authors = ["YOUR NAME <email@example.com>"]
+description = "OpenRelik worker to process Cloud provider logs."
+authors = ["roshan maskey <roshanmaskey@gmail.com>"]
 readme = "README.md"
 package-mode = false
 
 [tool.poetry.dependencies]
 python = "^3.10"
 celery = { extras = ["redis"], version = "^5.4.0" }
 openrelik-worker-common = "^0.10.0"
+orjson = "^3.10.16"
```

Comment:
I guess we are using this for speed sake?

Reply:
Yeah, using `orjson` improved performance.